phpBB/phpbb/captcha/plugins/qa.php

```diff
@@ -224,6 +224,11 @@ function get_template()
 
 		if ($this->is_solved() || empty($this->question_text) || !count($this->question_ids))
 		{
+			// don't log an error if the captcha has been solved
```

Comment:
Could the return value be stated, which will help for readability? Otherwise, a NULL return value will be used.

```
if ($this->is_solved())
{
		return false;
}
```